tests/test_checkpointing.py

```diff
@@ -164,57 +169,6 @@ def test_acoustic_save_and_nosave(shape=(50, 50), spacing=(15.0, 15.0), tn=500.,
     assert(np.allclose(rec.data, rec_bk))
 
 
-@silencio(log_level='WARNING')
-@skipif_yask
-@pytest.mark.parametrize('space_order', [4])
-@pytest.mark.parametrize('kernel', ['OT2'])
-@pytest.mark.parametrize('shape', [(70, 80), (50, 50, 50)])
-def test_checkpointed_vs_not_checkpointed(shape, kernel, space_order):
```

Comment:
why dropping tests?

Reply:
Is this now covered in ``test_gradient`` ?

Reply:
Yes it is